README.md

```diff
@@ -1,24 +1,32 @@
-# zkSync 2.0 stack for Hardhat
+# Welcome to the Hardhat zkSync plugins repository!
```

Comment:
It seems that I don't have access to that page... Can you send me image link and text here?

Reply:
Oops.. sure... here is the actual markdown



# zkSync 2.0: Welcome to zkSync Hardhat plugins repository

[![Logo](https://user-images.githubusercontent.com/8230135/215079996-46ec1c91-e65d-4adb-8d7a-f7eecf851858.svg)](https://zksync.io/)

zkSync 2.0 is a layer 2 rollup that uses zero-knowledge proofs to scale Ethereum without compromising on security or
decentralization. Since it's EVM compatible (Solidity/Vyper), 99% of Ethereum projects can redeploy without refactoring
or re-auditing a single line of code. zkSync 2.0 also uses an LLVM-based compiler that will eventually let developers
write smart contracts in C++, Rust and other popular languages.